eng/pipelines/dotnet-sqlclient-ci-core.yml

```diff
@@ -12,12 +12,17 @@ parameters: # parameters are shown up in ADO UI in a build queue time
 - name: targetFrameworks
   displayName: 'Target Frameworks on Windows'
   type: object
-  default: [net462, net6.0, net8.0]
+  default: [net462, net8.0]
 
 - name: targetFrameworksLinux
   displayName: 'Target Frameworks on Non-Windows'
   type: object
-  default: [net6.0, net8.0]
+  default: [net8.0]
+
+- name: netcoreVersionTestUtils
+  displayName: 'Netcore Version for Test Utilities'
+  type: object
+  default: net8.0
```

Comment:
This version is set here, but passed down through to the test util project compilation. Trying to keep this at the highest level possible so that it's easy to bump in the future without digging down in the individual jobs.